acstools/acszpt.py

```diff
@@ -375,6 +375,9 @@ def fetch(self):
 
             LOG.info('Parsing the response and formatting the results...')
             self._parse_and_format()
+            if self._warnings:
+                _ = [LOG.warning(w) for w in self._warnings]
```

Comment:
No need to construct a list.
```suggestion
            for w in self._warnings:
                LOG.warning(w)
```

Reply:
Thanks! This is a much better solution.

Reply:
Hmm. Did you forget to commit this suggestion?

Reply:
Why yes, yes I did. Committed! Thanks.